tests/DataFixtures/ORM/PromotionsFixture.php

```diff
@@ -22,57 +24,102 @@ public function getDependencies()
 
     public function load(ObjectManager $manager)
     {
+        /*
+        PROMOTION CONTEXTS:
+        ===================
+
+        Brand (b010t19z)
+            promotion 0 (p000000h, SUPER PROMOTION)
+
+            series 1 (b00swyx1)
+                promotion 1 (p000001h)
```

Comment:
Would it be worth having a super promo on the series or episode to prove that the inheritance is working correctly?